docs/static/releasenotes.asciidoc

```diff
@@ -52,6 +53,19 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-8-21]]
+=== Logstash 6.8.21 Release Notes
+
+[[notable-6-8-21]]
+==== Performance improvements and notable issues fixed
+
+* Add java version parameterization to Logstash builds https://github.com/elastic/logstash/pull/13384[#13384]
```

Comment:
This commit is an 'internal' one, only really relevant to CI, should we mark it as such, or skip it entirely from release notes?

Reply:
Removed :+1: